hosts.go

```diff
@@ -122,6 +122,8 @@ func (h *Host) DateStringFromCreatedAt() string {
 }
 
 // IPAddresses returns ipaddresses
+//
+// Deprecated: use Host#IPAddressesAll instead.
```

Comment:
`Host#IPAddresses` is widely used via `mkr` command (like `mkr hosts | jq -r '.[] | select(.ipAddresses | has("eth0")) | ...`), so it is difficult to make `Host#IPAddresses` deprecated.